protoc-gen-openapiv2/internal/genopenapi/template.go

```diff
@@ -793,7 +793,7 @@ func primitiveSchema(t descriptorpb.FieldDescriptorProto_Type) (ftype, format st
 		// NOTE: in OpenAPI specification, format should be empty on boolean type
 		return "boolean", "", true
 	case descriptorpb.FieldDescriptorProto_TYPE_STRING:
-		// NOTE: in OpenAPI specification, format should be empty on string type
+		// NOTE: in OpenAPI specification, can be empty on string type
```

Comment:
While we're here, can we add a reference to the data types part of the spec to the docstring of this function? This link: https://swagger.io/specification/v2/#data-types.